scripts/kuma/installer.sh

```diff
@@ -81,4 +81,5 @@ if ! rm -rf kuma-$KUMA_VERSION; then
 fi
 
 printf "INFO\tKuma service mesh has been installed successfully!!\n"
-printf "Visit: https://meshery.io/adapters/kuma/dashboard for more information\n"
+printf "Visit: https://meshery.io/adaptors/kuma/dashboard for more information\n"
```

Comment:
does this endpoint work? "dashboard"?

Reply:
Not yet lee, this is a quick feature once we agree upon certain good practices. Let's take this over a call.